src/anyvar/translate/variation_normalizer.py

```diff
@@ -19,48 +22,98 @@ def __init__(self, endpoint_uri: str):
 
         openapi_docs = self.endpoint_base + "openapi.json"
         resp = requests.get(openapi_docs)
-        if resp.status_code != 200:
+        if resp.status_code != HTTPStatus.OK:
             raise TranslatorConnectionException(
                 f"Failed to get response from Variation Normalizer REST endpoint at "
                 f"{openapi_docs}"
             )
 
-    def _send_rest_request(self, request_url: str):
+    def _send_rest_request(self, request_url: str) -> requests.Response:
         """Emit normalization request. Broken out to enable mocking.
 
         :param request_url: URL containing normalization request parameters
+        :return: content of response
         """
         return requests.get(request_url)
 
-    def translate(self, var: str, **kwargs: Dict) -> vrs_models.Allele:
+    def get_sequence_id(self, accession_id: str) -> str:
+        """Get GA4GH sequence identifier for provided accession ID
+
+        :param accession_id: ID to convert
+        :return: equivalent GA4GH sequence ID
+        :raise: KeyError if no equivalent ID is available
+        """
+        req_url = self.endpoint_base + f"translate_identifier?identifier={accession_id}&target_namespaces=ga4gh"  # noqa: E501
+        resp = self._send_rest_request(req_url)
+        if resp.status_code == HTTPStatus.NOT_FOUND:
+            raise TranslatorConnectionException(
+                f"Failed to get response from Variation Normalizer REST endpoint at {req_url}"  # noqa: E501
+            )
+        elif resp.status_code == HTTPStatus.INTERNAL_SERVER_ERROR:
+            raise TranslatorConnectionException(
+                f"Variation Normalizer REST endpoint returned server error for {accession_id}"  # noqa: E501
+            )
+        elif resp.status_code != HTTPStatus.OK:
+            raise TranslatorConnectionException(
+                f"Variation Normalizer REST endpoint returned {resp.status_code} for {accession_id}"  # noqa: E501
+            )
+
+        resp_json = resp.json()
+        if resp_json.get("warnings") or len(resp_json.get("aliases", [])) == 0:
+            raise KeyError(f"Unable to find GA4GH ID for {accession_id}")
+        return resp_json["aliases"][0]
+
+    @staticmethod
+    def _safe_check_variation_type(var_normalizer_response: Dict) -> Optional[str]:
+        """Safely check variation type in response from variation normalizer. Resilient
+        to different response structures.
+
+        :param var_normalizer_response: complete response received from normalizer
+        :return: type value if exists, None otherwise
+        """
+        variation_descriptor = var_normalizer_response.get("variation_descriptor")
+        if not variation_descriptor:
+            return None
+        variation = variation_descriptor.get("variation")
+        if not variation:
+            return None
+        return variation.get("type")
+
+    def translate(self, var: str, **kwargs: Dict) -> Optional[VrsPythonVariation]:
```

Comment:
This is a little nitpicky, but Variation Normalizer does not return `Haplotype` or `VariationSet` which are in `VrsPythonVariation`. Variation Normalizer does return `AbsoluteCopyNumber` and `RelativeCopyNumber` which are not in `VrsPythonVariation`. It might make sense to have a separate `VariationNormalizerVariation` type?

Reply:
Yeah this is dangerous territory. Theoretically our scope here is supposed to start with VRS 1.2 and move forward to 1.3 when able, so I'm going to hold off on anything copy number related until VRS-Python is updated and the 1.3 PR for variation normalizer goes through